swing/src/main/java/info/openrocket/swing/gui/simulation/SimulationConditionsPanel.java

```diff
@@ -193,10 +197,10 @@ public void stateChanged(EventObject e) {
 
 		m = new DoubleModel(target, "LaunchLatitude", UnitGroup.UNITS_LATITUDE, -90, 90);
 
-		spin = new JSpinner(m.getSpinnerModel());
-		spin.setEditor(new SpinnerEditor(spin));
-		spin.setToolTipText(tip);
-		sub.add(spin, "growx");
+		var latitudeSpinner = new JSpinner(m.getSpinnerModel());
```

Comment:
Please don't use `var` types; it serves no purpose here and I generally don't like using it in Java.